packages/dsh-tauri-panel-extension/package.json

```
@@ -0,0 +1,76 @@
+    "dev": "tsdown --watch",
+    "prepublishOnly": "nr build",
+    "start": "tsx src/index.ts",
+    "typecheck": "tsc"
+  },
+  "dependencies": {
+    "dsh-tauri": "workspace:*",
+    "smol-toml": "catalog:utils",
+    "yaml": "catalog:testing"
+  },
+  "devDependencies": {
+    "dompurify": "catalog:ui",
```

ESLint

[error] Expected object keys to be in ascending order. 'dompurify' should be after '`@deepseek-ai/cordis`'.

(jsonc/sort-keys)